kafka-client/src/main/java/dev/responsive/kafka/store/StoreUtils.java

```diff
@@ -0,0 +1,22 @@
+package dev.responsive.kafka.store;
+
+import java.time.Duration;
+
+public class StoreUtils {
```

Comment:
I added a static StoreUtil class under utils with one method. Could you move that method here (or vice versa)?